CRM/Mailing/Info.php

```diff
@@ -83,7 +83,12 @@ public static function createAngularSettings():array {
     $enabledLanguages = CRM_Core_I18n::languages(TRUE);
     $isMultiLingual = (count($enabledLanguages) > 1);
     $requiredTokens = Civi\Core\Resolver::singleton()->call('call://civi_flexmailer_required_tokens/getRequiredTokens', []);
-
+    $default_email = \Civi\Api4\Email::get(TRUE)
+      ->addWhere('contact_id', '=', 'user_contact_id')
+      ->addWhere('is_primary', '=', '')
```

Comment:
@stesi561 why this filter? ie why not is_primary = TRUE?

Reply:
Hmm. How did that happen. I swear that is what I'd set up in API explorer before copying across.

Reply:
Fixing.